src/Client/Utils/A.php

```diff
@@ -50,4 +50,12 @@ public static function fetchRequest()
     {
         return RequestBuilder::create('fetch');
     }
-}
+
+    /**
+     * @return \Mcustiel\Phiremock\Client\Utils\RequestBuilder
+     */
+    public static function deleteRequest()
+    {
+        return RequestBuilder::create('delete');
+    }
+}
```

Comment:
Please add a blank line at the end of the file and we are ready to merge. Never saw this one was missing, thank you.